src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Hello = function(props) {
```

Comment:
In a case where we're declaring and initializing a react component, it might be a bad idea to use arrow syntax. Even if it works we might be sacrificing readability for concision.